### ISSUE TITLE
Does the SC 1.4.3 Contrast minium exception apply to text outside a disabled control?

### ISSUE THREAD

**Issue by jha11y** [w3c/wcag3#160]
For the "Text or images of text that are part of an inactive user interface component" exception for WCAG 1.4.3 Contrast (Minimum), does the exception apply to the label for a control when it is outside of that control?

For example, a radio button that is disabled, does the text label does have to meet the contrast requirements of SC 1.4.3?

Would the exception apply to "helper text" (text underneath a text input field)? Error messages positioned like helper text?

I did check the other WCAG GitHub issues to see if my question had been answered, but I did not find this exact question.  I know text inside a disabled control (e.g., HTML button)  would be exempted.

**Comment by Myndex** [comment]
I question if this is relevant for WCAG3. 

With a perceptually uniform contrast metric, nuanced elements should be covered, meaning "disabled" controls should have guidance in terms of being at least spot-readable, but also being lower contrast or otherwise visibly indicated different than active controls.

1.4.3 in WCAG 2 does none of this, but WCAG 3 should avail itself of more complete visual appearance guidance.

**Comment by mbgower** [comment]
@Myndex you first and last statements seem contradictory. 

If "WCAG 3 should avail itself of more complete visual appearance guidance", then it would seem a discussion concerning considerations for disabled and read-only components is relevant.

**Comment by mraccess77** [comment]
The language of 1.4.3 seems to indicate that the text label of an input that is inactive would not need to meet contrast requirements as it's part of the inactive UI component/control.

Regarding read only controls - I personally don't consider those to be inactive and thus I think contrast would still apply - but I agree it's not clear from the normative wording.

**Comment by Myndex** [comment]
> [@Myndex](https://github.com/Myndex) you first and last statements seem contradictory.
> 
> If "WCAG 3 should avail itself of more complete visual appearance guidance", then it would seem a discussion concerning considerations for disabled and read-only components is relevant to that goal.

Hi @mbgower 

I see what you mean, that was unintentional, editing for clarity. Since this thread was moved from WCAG, I meant that the relevance for disabled elements is somewhat moot there, as excluded.

IMO what should be included is all disabled elements, or at least, disabled elements that identify the element when it's visible in its disabled mode. This aspect is very relevant to WCAG 3——but not just for text outside a control.
